src/plugins/intel_gna/src/gna_graph_compiler.cpp

```diff
@@ -736,6 +764,8 @@ void GNAGraphCompiler::finalizeConvolution2DPrimitive(InferenceEngine::CNNLayerP
     const auto weightPrec = OvGnaTypeIntFromBytes(convolution._weights->getTensorDesc().getPrecision().size());
     const auto biasPrec = OvGnaTypeIntFromBytes(biasPrecision.size());
 
+    const auto is_dwsc = (convolution._group > 1);
```

Comment:
maybe it would be good to create common method for checking if convolution is dwsc? 
this condition is used at least in two places.